### PR TITLE
mpv.profile: add new mpv watch_later path

### PR DESCRIPTION
**Pull request by CodeWithMa**
The new version of mpv changed the path of the watch_later folder. I added the new path to mpv.profile.

See https://github.com/mpv-player/mpv/pull/10838 for the corresponding mpv change.

### CODE REVIEW

File: etc/profile-m-z/mpv.profile [hunk 1]
```diff
@@ -55,6 +56,7 @@ whitelist ${HOME}/.config/mpv
 whitelist ${HOME}/.config/youtube-dl
 whitelist ${HOME}/.config/yt-dlp
 whitelist ${HOME}/.config/yt-dlp.conf
+whitelist ${HOME}/.local/state/mpv/watch_later
```
Comment (glitsj16) [inline etc/profile-m-z/mpv.profile:59]: If we're going for the ~/.local/share/state/mpv path when (no)blacklisting, this needs to be adjusted accordingly.

**Review by glitsj16 — CHANGES_REQUESTED** [review]
Add the blacklist counterpart. Otherwise noblacklist won't work as expected.

File: etc/profile-m-z/mpv.profile [hunk 2]
```diff
@@ -28,6 +28,7 @@ noblacklist ${HOME}/.config/mpv
 noblacklist ${HOME}/.config/youtube-dl
 noblacklist ${HOME}/.config/yt-dlp
 noblacklist ${HOME}/.config/yt-dlp.conf
+noblacklist ${HOME}/.local/state/mpv/watch_later
```
[the remark this answers is not in the document]
Comment (CodeWithMa) [reply etc/profile-m-z/mpv.profile]: blacklist ${HOME}/.local/state/mpv added to disable-programs.inc

File: etc/profile-m-z/mpv.profile (hunk 1 above)
Comment (CodeWithMa) [reply etc/profile-m-z/mpv.profile]: adjusted to `whitelist ${HOME}/.local/state/mpv`

**Review by glitsj16 — APPROVED** [review]
Thanks for implementing the changes! LGTM

File: etc/profile-m-z/mpv.profile [hunk 3]
```diff
@@ -55,6 +56,7 @@ whitelist ${HOME}/.config/mpv
 whitelist ${HOME}/.config/youtube-dl
 whitelist ${HOME}/.config/yt-dlp
 whitelist ${HOME}/.config/yt-dlp.conf
+whitelist ${HOME}/.local/state/mpv
```
Comment (rusty-snake) [inline etc/profile-m-z/mpv.profile:59]: No mkdir?

**Comment by kmk3** [comment]
Sorry, I accidentally pushed our master to the PR branch (as it's also named
master) and it closed the PR.

But the changes are merged now at commit 5e0f35b0c ("mpv.profile: add new
XDG_STATE_HOME path", 2023-08-02).

**Pull request closed without merging**